tox.ini

```diff
@@ -8,23 +8,20 @@
 skipsdist = true
 envlist = py{26,27,33,34,35},py{26,27}-oldest,cover,lint
 
-# nosetest -v => more verbose output, allows to detect busy waiting
-# loops, especially on Travis
```

Comment:
Replacing `nosetests` with `python setup.py test` allowed us to move mock to `test_requires`; setuptools then installs it on demand, when tests are run. That was the immediate practical benefit. However, since you've argued for not moving mock yet, there's less reason to do this at the moment. If you like, I can bundle it with the later PR that moves mock.